mobile/library/common/http/client.cc

```diff
@@ -124,26 +125,35 @@ void Client::DirectStreamCallbacks::encodeData(Buffer::Instance& data, bool end_
     response_data_->setWatermarks(1000000);
   }
 
-  // Send data if in default flow control mode, or if resumeData has been called in explicit
-  // flow control mode.
+  // Try to send data if
+  //  1) no min chunk size and in default flow control mode
```

Comment:
Based on what I think is happening in sendDataToBridge, is it more appropriate to say "try to send data if (1) in default flow control mode and either no min chunk size *or* bytes to send is greater than the chunk size"?